### DISCUSSION TITLE
How to run offline?

### DISCUSSION THREAD

**Discussion by CharlieJV** [boxgaming/qbjs#12] [Q&A]
Maybe a silly question, but I haven't figured out how to go about it.

What's involved in getting QBJS working offline?

**Comment by boxgaming** [comment]
Hey Charlie, just download the latest qbjs.zip file from the current release [here](https://github.com/boxgaming/qbjs/releases/download/v0.3.0-beta/qbjs.zip).  Unzip to your preferred location and open the index.html file to run.

**Reply by CharlieJV** [reply]
Did I miss that link somewhere in your github project pages?

That was easy to download and setup/run, but it should be easy to find and download.

You've got to set that up in your wiki page, because it is pretty slick.

**Reply by boxgaming** [reply]
Good suggestion, yes, the wiki home page definitely needs a bit more content.

**Comment by CharlieJV** [comment]
Hey, this isn't working offline.  How do I go about getting this to work without a connection to the web?

**Comment by boxgaming** [comment]
Hrm, looks like the 0.3.0-beta still had some JS library dependencies that were referencing online resources.  So yes, I'm afraid that version will not work completely disconnected from the web.  The dev build has all library dependencies included.  This will be available in the 0.4.0-beta release.  Or, if you'd like to check it out early you could download straight from the current github dev version: [here](https://github.com/boxgaming/qbjs/archive/refs/heads/main.zip).

**Reply by CharlieJV** [reply]
That works A-1 offline.  I imagine you know this already, but fyi just in case: 0.4.0-beta breaks the "share" feature.

**Reply by boxgaming** [reply]
That's the only risk with pulling the latest dev release, things could be broken due to changes in progress.   If you download the latest dev version with the same link above it should be working again.
